bobtemplates/plone_addon/.travis.yml.bob

```diff
@@ -1,22 +1,28 @@
 language: python
 sudo: false
 cache:
-  pip: true
   directories:
-    - $HOME/buildout-cache
+  - eggs
+  - downloads
 python:
   - 2.7
+matrix:
+  fast_finish: true
 install:
-  - python bootstrap-buildout.py -c travis.cfg
-  - bin/buildout -Nc travis.cfg buildout:eggs-directory=$HOME/buildout-cache/eggs buildout:download-cache=$HOME/buildout-cache/downloads
+  - python bootstrap-buildout.py
+  - bin/buildout annotate
+  - bin/buildout buildout:download-cache=downloads
```

Comment:
This should be (as it was before) ``bin/buildout -N buildout:download-cache=downloads`` to get most out of caching.